plugins/SimulationView/SimulationViewMenuComponent.qml

```diff
@@ -216,15 +211,14 @@ Item
                 onPreferenceChanged:
                 {
                     layerTypeCombobox.currentIndex = UM.SimulationView.compatibilityMode ? 1 : UM.Preferences.getValue("layerview/layer_view_type");
-                    layerTypeCombobox.updateLegends(layerTypeCombobox.currentIndex);
-                    playButton.pauseSimulation();
```

Comment:
We no longer pause the simulation when a preference is changed? Why was this changed?

I think this was in there because we want the simulation to pause when changing the colour scheme of the simulation.

Reply:
This was a mistake on my end. The pause button got moved to a different QML file. As it gave warnings, i removed that line, but forgot to re-add it.